bootstrap.js

```diff
@@ -14,9 +14,9 @@ function startup(data, reason) {
 function shutdown(data, reason) {
   console.log('bootstrap shutdown called');
   // no teardown is needed for a normal shutdown
-  if (reason == APP_SHUTDOWN) { return; }
+  if (reason === APP_SHUTDOWN) { return; }
 
-  if (reason == ADDON_DISABLE || reason == ADDON_UNINSTALL) {
+  if (reason === ADDON_DISABLE || reason === ADDON_UNINSTALL) {
```

Comment:
Yeah, this is a bit weird for sure. Welcome to XUL-landia.

The `reason`s are defined by the code that runs the bootstrap file, such that the constants should be predefined when the code executes: https://developer.mozilla.org/en-US/Add-ons/Bootstrapped_extensions#Reason_constants